packages/vscode-dependencies-validation/nyc.config.js

```diff
@@ -3,5 +3,5 @@ module.exports = {
   // No integration or end to end tests currently in this project.
   // `extension.ts` only contains minimal integration code with VSCode.
   // and can be "safely" excluded.
-  exclude: ["src/extension.ts", "src/commands.ts"],
+  exclude: ["src/extension.ts", "src/commands.ts", "src/logger/logger.ts"],
```

Comment:
you may also need to add exclusions for the **root** nyc.config 
so it won't fail when it does the coverage merging

Reply:
Done.